src/pages/exercise2.jsx

```diff
@@ -0,0 +1,7 @@
+const exercise2 = () => {
```

Comment:
Since this component is not being used anywhere, it would be better to remove it.